Src/ILGPU/Runtime/OpenCL/CLAccelerator.cs

```diff
@@ -318,7 +318,7 @@ protected override CLKernel CreateKernel(CLCompiledKernel compiledKernel)
 
         /// <inheritdoc/>
         protected override AcceleratorStream CreateStreamInternal() =>
-            new CLStream(this);
+            new CLStream(this, Context.Properties.EnableProfiling);
```

Comment:
Question: can we check this flag in all cases to ensure consistent behavior for all GPUs?

Reply:
OK, will update.